src/Escaper/Escaper.c

```diff
@@ -49,6 +48,30 @@ Escaper_start_decoder(Escaper* const self)
     self->m_parse_state = Escaper_State_Wait;
 }
 
+void
+Escaper_decode_escape_byte(Escaper* const self, const uint8_t byte)
```

Comment:
I don't see previous declaration of this function. Either add proper declaration to h, or possibly better, make this function local: static (inline?) and without "Escaper_" prefix.

Reply:
My intention was to make this function local/private. It isn't a part of API, but it is a helper. 
Sure, I'll remove the prefix and add static keyword then.

Reply:
I consider changing first parameter's name from 'self' to 'escaper' then, what do you think?

Reply:
Why change the convention?